docs/reference/commandline/build.md

```diff
@@ -277,7 +277,7 @@ flag](../run.md#specifying-custom-cgroups).
 
 Using the `--ulimit` option with `docker build` will cause each build step's
 container to be started using those [`--ulimit`
-flag values](../run.md#setting-ulimits-in-a-container).
+flag values](./run.md#set-ulimits-in-container-ulimit).
```

Comment:
Yup, GitHub uses a slightly different method for generating those. We should look into HUGO if we can change that to match GitHub